.github/workflows/CI.yaml

```diff
@@ -36,4 +36,4 @@ jobs:
 
     - run: poetry run mypy .
 
-    - run: poetry run pytest --cov
\ No newline at end of file
+    - run: poetry run pytest tests/unit --cov
```

Comment:
@igiloh-pinecone as a start we just run unit tests in CI and system tests manually. In the future we should run also system tests on merges

Reply:
@acatav why? Is it a problem of permissions? 

If it's simply an optimization of "time to merge a PR" - I would say ditch it. Let all the full tests run on every PR.  
Our current CI runs are about 7 minutes, about third of that just `poetry install` - which BTW we should optimize with caching, like we did in `Canopy`. So we can quickly reduce it to 5 min.   
I don't see a reason to optimize it further than that. Let's keep running the full test suite on every PR.

Reply:
done